subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -105,28 +136,22 @@ int fota_download_start(char *host, char *file)
 	}
 
 	/* Verify that a download is not already ongoing */
-	if (dfu.fd != -1) {
+	if (dlc.fd != -1) {
```

Comment:
`download_client_connect()` does nothing and returns success if already connected, you don't need to check this.

Reply:
fixed